File: src/services/Hunting/Modules/Document/Modules.Document.Core/Entities/Raid.cs
using Modules.Document.Domain.Interfaces;

namespace Modules.Document.Domain.Entities
{
    public class Raid : IBaseEntity
    {
        public Guid Id { get; set; }
        public DateTime ExitTime { get; set; }
        public DateTime ReturnedTime { get; set; }
        public ICollection<User> Participants { get; set; } = [];
        public string Note { get; set; } = string.Empty;
        public Guid LandId { get; set; }
        public Land? Land { get; set; }
    }
}

File: src/uis/ReactWeb/src/entities/raid/Raid.ts
import {Land} from '@/entities/land/Land.ts';
import {User} from '@/entities/user/models/User.ts';

export interface Raid {
    id?:string;
    exitTime: Date;
    returnedTime: Date;
    participants: User[];
    note:string;
    landId: string;
    land:Land;
    status: string;
}